### PR TITLE
RHBZ#2177673: fwcfg64: don't fail if unable to work with vmcoreinfo

### PR DESCRIPTION
**Pull request by viktor-prutyanov**
Previously, fwcfg64 driver was failing if fw_cfg device was present, but `etc/vmcoreinfo` entry wasn't. The driver shouldn't fail when device is OK. Anyway, `dump-guest-memory -w` will not work without vmcoreinfo.

### CODE REVIEW

File: fwcfg64/power.c [hunk 1]
```diff
@@ -120,13 +123,20 @@ NTSTATUS FwCfgEvtDevicePrepareHardware(IN WDFDEVICE Device,
                 "VMCoreInfo index is 0x%x", ctx->index);
     if (!ctx->index)
     {
-        return STATUS_BAD_DATA;
+        TraceEvents(TRACE_LEVEL_VERBOSE, DBG_PNP,
```
Comment (YanVugenfirer) [inline fwcfg64/power.c:126]: Will the line 110 not fail?    
if (FWCfgCheckSig(ctx->ioBase) ||
        FWCfgCheckFeatures(ctx->ioBase, FW_CFG_VERSION_DMA) ||
        FWCfgCheckDma(ctx->ioBase))
    {
        return STATUS_BAD_DATA;
    }

Comment (viktor-prutyanov) [reply fwcfg64/power.c:126]: @YanVugenfirer  It will not fail if fw_cfg is peresent

Comment (viktor-prutyanov) [reply fwcfg64/power.c:126]: On modern QEMU it can't fail

Comment (viktor-prutyanov) [reply fwcfg64/power.c:126]: Some very old versions can't do "DMA"

**Comment by viktor-prutyanov** [comment]
rerun tests

**Review by YanVugenfirer — APPROVED** [review]

**Comment by kostyanf14** [comment]
rerun tests

**Pull request merged**